laurent22/joplin
# Desktop : fa-icons are not rendered in Mermaid Diagram

This PR fixes #8914  with Mermaid code blocks in the Markdown Pane / Read-Only Preview 

Changes Made:

Added the code to properly render fa-fa icons in both the Rich Text Editor and Markdown Pane/Read-Only Mode
Ensured that the graph still renders correctly without any errors or issues

Testing:

https://github.com/user-attachments/assets/6f21fc16-2d95-4ee1-a46e-686945876783

Steps to Test:

Create a new note 
Add a code block with language mermaid
Set the following content:

```
 mindmap
      root((mindmap))
          Long history
          ::icon(fa fa-book)
```
Click "Ok" to render the graph
Switch between editors to check whether the icons are rendered or not
Let me know if any further changes or information are needed.
